tinnsleep/classification.py

```diff
@@ -59,6 +64,34 @@ def fit(self, X, y=[], sample_weight=None):
         # Return the classifier
         return self
 
+    def partial_fit(self, X, y=[]):
+        """Updating average amplitude for adaptive thresholding
+
+                Parameters
+                ----------
+                X : {array-like} of shape (n_samples, n_features)
+
+                y : Ignored
+                    Not used, present here for API consistency by convention.
+
+                sample_weight : array-like of shape (n_samples,), default=None
+                    The weights for each observation in X. If None, all observations
+                    are assigned equal weight.
```

Comment:
Cited in docstring but not put as an input of the function.

Reply:
thanks !